spec/controllers/doorkeeper/custom_applications_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Doorkeeper::CustomApplicationsController, type: :controller do
```

Comment:
The other permission tests are implemented as feature specs - can we change this to a feature spec so it is consistent please?